src/pages/ProfilePage.jsx

```diff
@@ -76,17 +83,18 @@ function ProfilePage(props) {
                             <h2 className="fw-medium fs-5 m-0">Edite e adicione informações sobre você</h2>
```

Comment:
```suggestion
                            <h6 className="fw-medium m-0">Edite e adicione informações sobre você</h6>
```
- Eu particularmente prefiro assim porque o texto não fica com quebra de linha (vide figma), mas dá uma olhada, se tu achar pequeno demais, basta trocar h6 por h5 e assim a gente nem precisa adicionar fs-something :D
- Na div acima "\<h1>...\</h1>", sou mais fã do pb-0 também (por conta do que está no figma)... Mas fica a teu critério daí :D

Reply:
nossa pq que tinha um fs-5 ali kkkkkkkkkkkkkkkkkk bem reparado mano, não tinha percebido